homeassistant/components/geniushub/climate.py

```diff
@@ -0,0 +1,204 @@
+"""
+Supports Genius hub to provide climate controls.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/climate.geniushub/
```

Comment:
We don't include the url in the docstring anymore. Just keep the very first line.